src/Command/SiteDbImportCommand.php

```diff
@@ -90,6 +103,11 @@ protected function execute(InputInterface $input, OutputInterface $output) {
       throw new SiteCommandException('Please specify a file to import the dump from');
     }
 
+    // If the db dump is not local, download it locally.
+    if (!stream_is_local($this->filename)) {
```

Comment:
Maybe we need some extra check for the existence of the file (in case of nfs mount) and use the stream wrapper to check the file size to avoid copying it again if it the size did not change since last execution